package.json

```diff
@@ -106,7 +106,8 @@
     "stylelint": "9.9.0",
     "stylelint-config-wordpress": "13.1.0",
     "webpack": "4.28.3",
-    "webpack-cli": "3.1.2"
+    "webpack-cli": "3.1.2",
+    "webpack-rtl-plugin": "github:yoavf/webpack-rtl-plugin#develop"
```

Comment:
Should we target `master` ?

Reply:
I chose that one to keep consistency with Gutenberg:
https://github.com/WordPress/gutenberg/blob/master/package.json#L123

Ideally, we would be using `webpack-rtl-plugin` from NPM, but we need this PR to be merged upstream before: https://github.com/romainberger/webpack-rtl-plugin/pull/15.